controllers/comments-controllers.js

```diff
@@ -16,3 +19,20 @@ exports.getCommentsByArticleId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postCommentByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const { body } = req;
+  const proms = [
+    insertCommentByArticleId(body, article_id),
+    checkExists("articles", "article_id", article_id),
```

Comment:
You may find that you don't actually need to check if the article exists for this one.
As the article id provided into the comments table is a FOREIGN KEY and must be present in the articles table, PSQL will generate an error for us if this is not the case